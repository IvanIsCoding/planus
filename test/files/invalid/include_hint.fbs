include "helpers/include_hint_helper1.fbs";
include "helpers/include_hint_helper2.fbs";