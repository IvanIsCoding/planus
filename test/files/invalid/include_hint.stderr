[0m[1m[38;5;9merror[0m[1m: Unknown type[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/helpers/include_hint_helper1.fbs:2:6
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: [0m[31mBar[0m;
  [0m[34mâ”‚[0m      [0m[31m^^^[0m
  [0m[34mâ”‚[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/helpers/include_hint_helper2.fbs:1:1
  [0m[34mâ”‚[0m
[0m[34m1[0m [0m[34mâ”‚[0m table Bar {}
  [0m[34mâ”‚[0m [0m[34m---------[0m [0m[34mPerhaps you meant to include test/files/invalid/helpers/include_hint_helper2.fbs[0m

Error: Bailing because of errors
